.github/workflows/docker.yml

```diff
@@ -35,6 +42,15 @@ jobs:
           password: "${{ secrets.DockerPassword }}"
           build_extra_args: "--build-arg=ENVIRONMENT=deployment"
 
+      - name: Build and Push unstable Docker image tag
+        uses: whoan/docker-build-with-cache-action@v4
+        with:
+          image_name: ${{ env.IMAGE_NAME }}
+          username: gadockersvc
+          image_tag: ${{ env.UNSTABLE_TAG }}
+          password: "${{ secrets.DockerPassword }}"
+          build_extra_args: "--build-arg=ENVIRONMENT=deployment"
+
       - name: Build and Push latest Docker image (old name)
```

Comment:
Let's get rid of the old docker image... moving forwards!

Reply:
Too many changes I am not sure I have changed them all in all my environments. Delayed deprecation.